pkg/discovery/globalnet/config_map.go

```diff
@@ -31,11 +31,11 @@ import (
 )
 
 const (
-	globalCIDRConfigMapName     = "submariner-globalnet-info"
-	globalnetEnabledKey         = "globalnetEnabled"
-	clusterInfoKey              = "clusterinfo"
-	globalnetCidrRange          = "globalnetCidrRange"
-	globalnetClusterSize        = "globalnetClusterSize"
+	GlobalCIDRConfigMapName     = "submariner-globalnet-info"
+	GlobalnetEnabledKey         = "globalnetEnabled"
+	ClusterInfoKey              = "clusterinfo"
+	GlobalnetCidrRange          = "globalnetCidrRange"
+	GlobalnetClusterSize        = "globalnetClusterSize"
```

Comment:
Why are we exporting these?

Reply:
Right, I saw that (which I should have mentioned), but I don’t think it’s great to have UTs in another project relying on internals of the operator and Globalnet.

Reply:
> I really have two questions. (a) Do we want to have a UT for this in the addon?

I think we want to have the addon test/verify it correctly processed/configured Globalnet. Internally it uses operator functionality that has to be setup in the tests.

 (b) If we do, shouldn’t we encapsulate things better in the operator for the addon? For example, add a constructor.

yeah I think we should be able to do something like that to avoid exposing the internal config map keys. 

@Jaanki I have cycles to look into that if you want.

Reply:
@tpantelis sure !!

Reply:
It turns out there's already `CreateGlobalnetConfigMap` and `NewGlobalnetConfigMap` functions that encapsulate 
 the constants. I pushed [this addon PR](https://github.com/stolostron/submariner-addon/pull/384) to use them. The only remaining reference is [here](https://github.com/stolostron/submariner-addon/blob/81e24d31bc6e5ded7fd7b658e43e64b5b37def3e/pkg/hub/submarineragent/controller_test.go#L565) and we can simply add a wrapper function:

```Go
func DeleteGlobalnetConfigMap(kubeClient kubernetes.Interface, namespace string) error {
	return kubeClient.CoreV1().ConfigMaps(namespace).Delete(context.TODO(), GlobalCIDRConfigMapName, metav1.GetOptions{})
}
```

to take care of that. I'll push a commit to this PR to do that.